MediaRecorder.html

```diff
@@ -327,17 +361,22 @@
           <dt><dfn><code>stop</code></dfn></dt>
           <dd>
             When a <code>MediaRecorder</code> object&#8217;s <code>stop</code>
-            method is invoked, the UA MUST queue a task, using the DOM
-            manipulation task source, that runs the following steps:
+            method is invoked, the UA MUST run the following steps:
             <ol>
-              <li>If <code>state</code> is <code>inactive</code>, throw an
-              <code>InvalidStateError</code> DOMException and terminate these
-              steps. Otherwise:</li>
-              <li>Set <code>state</code> to <code>inactive</code> and stop
-              gathering data.</li>
-              <li>Fire an event named <code>dataavailable</code> containing the
-              Blob of data that has been gathered.</li>
-              <li>Fire an event named <code>stop</code>.</li>
+              <li>If <a>state</a> is <a href="#dom-recordingstate-inactive">
+              inactive</a>, throw an <code>InvalidStateError</code> DOMException
+              and terminate these steps. Otherwise the UA MUST queue a task,
+              using the DOM manipulation task source, that runs the following
+              steps:</li>
+              <li>Set <a>state</a> to <a href="#dom-recordingstate-inactive">
```

Comment:
Nit: maybe s/following/remaining/ ? Otherwise I'm used to seeing queued steps indented one level and renumbered. Maybe just be me. Or maybe hoist first step into initial paragraph, dunno.